lib/Doctrine/Common/DataFixtures/ReferenceRepository.php

```diff
@@ -385,11 +384,7 @@ public function getManager()
      */
     protected function getRealClass($className)
     {
-        if (substr($className, -5) === 'Proxy') {
-            return substr($className, 0, -5);
-        }
-
-        return $className;
+        return $this->manager->getMetadataFactory()->getMetadataFor($className)->getName();
```

Comment:
```suggestion
        return $this->manager->getClassMetadata($className)->getName();
```
This would allow to simplify tests I think

Reply:
Thanks, done.